models/Airplane/Airplane.jl

```diff
@@ -146,51 +156,96 @@ const Ixz = 0.0  # not used (terms are simplified)
     dx[18] = zero(Mz)
 end;
 
-controller = read_nnet(@modelpath("Airplane", "controller_airplane.nnet"));
-
 # ## Specifications
-#
-# ``x_2`` should be in ``[−0.5, 0.5]`` and ``x_7, x_8, x_9`` should be in ``[-1.0,1.0]``.
+
+# The safety property is that ``x_2`` should be in ``[−0.5, 0.5]`` and
+# ``x_7, x_8, x_9`` should be in ``[-1.0, 1.0]``.
+
+controller = read_nnet(@modelpath("Airplane", "controller_airplane.nnet"))
 
 X₀ = Hyperrectangle(low=[0.0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-                    high=[0.0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0]);
-U₀ = ZeroSet(6);
+                    high=[0.0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0])
+if falsification
+    ## choose a single point in the initial states (here: the top-most one)
+    X₀ = Singleton(high(X₀))
+end
+U₀ = ZeroSet(6)
 
-vars_idx = Dict(:state_vars=>1:12, :control_vars=>13:18);
-ivp = @ivp(x' = airplane!(x), dim: 18, x(0) ∈ X₀ × U₀);
+vars_idx = Dict(:state_vars=>1:12, :control_vars=>13:18)
+ivp = @ivp(x' = airplane!(x), dim: 18, x(0) ∈ X₀ × U₀)
 
-period = 0.1;  # control period
-T = 20 * period;  # time horizon
+period = 0.1  # control period
 
 prob = ControlledPlant(ivp, controller, vars_idx, period);
 
-safe_states = HPolyhedron([
-    HalfSpace(SingleEntryVector(2, 18, 1.0), 0.5),
-    HalfSpace(SingleEntryVector(2, 18, -1.0), 0.5),
-    HalfSpace(SingleEntryVector(7, 18, 1.0), 1.0),
-    HalfSpace(SingleEntryVector(7, 18, -1.0), 1.0),
-    HalfSpace(SingleEntryVector(8, 18, 1.0), 1.0),
-    HalfSpace(SingleEntryVector(8, 18, -1.0), 1.0),
-    HalfSpace(SingleEntryVector(9, 18, 1.0), 1.0),
-    HalfSpace(SingleEntryVector(9, 18, -1.0), 1.0)
-   ]);
-# TODO spec: x[2] ∈ [−0.5, 0.5] and [x[7], x[8], x[9]] ∈ [-1, 1]^3 for all t
+# Safety specification: ``x[2] ∈ [−0.5, 0.5] & [x[7], x[8], x[9]] ∈ [-1, 1]^3``
```

Comment:
this line was not being displayed properly but i think you already fixed it

Reply:
I am not sure what "displayed properly" means. If it is wrong in the notebook output, then we should fix it.

Reply:
I think I see now. When using `$` instead of double backticks, it displays as math. I just fixed that.